frontend/src/routes/index.tsx

```diff
@@ -0,0 +1,62 @@
+import ButtonLink from '@/components/general/links/ButtonLink'
+import IntroductionSlider from '@/components/startpage/IntroductionSlider'
+import KeyFacts from '@/components/startpage/KeyFacts'
+import { createFileRoute } from '@tanstack/react-router'
+import { Mail, MoveRight } from 'lucide-react'
+import Lottie from 'lottie-react';
+import dashboardAnimation from '../../src/animations/dashboardAnimation.json';
+import QuickLinks from '@/components/startpage/QuickLinks'
+import useStore from '@/store/store'
+
+export const Route = createFileRoute('/')({
+  component: Startpage,
+})
+
+function Startpage() {
+  const store = useStore.getState();
```

Comment:
Change the store here too, as I mentioned before

```suggestion
  const { isAuthenticated } = useStore((state) => ({
    isAuthenticated: state.auth.isAuthenticated,
  }));
```